features/02_configure_aruba/basics.feature

```diff
@@ -55,7 +55,7 @@ Feature: Usage of configuration
     Given a file named "spec/support/aruba_config.rb" with:
     """ruby
     Aruba.configure do |config|
-      config.exit_timeout = 0.5
+      config.exit_timeout = 0.7
```

Comment:
If I remember correctly, I increased the timeouts so that the tests would pass the `test-windows (3.0)` GitHub Action. The log from that run isn't showing up anymore when I [take a look at it](https://github.com/cucumber/aruba/actions/runs/4634968300/jobs/8206847410), so I'm not able to confirm that. I'll take out the changes and see what happens.

Reply:
It's also worth noting, that `simplecov` takes some time to start, so that could throw the timing off.

Reply:
Hm .. maybe `aruba-test-cli` inherits `RUBYOPT` and therefore loads `simplecov_setup`? That shouldn't happen...